### ISSUE TITLE
Picker 2.6.x transforms numeric values to strings

### ISSUE THREAD

**Issue by swrobel** [react-native-picker/picker#538]
If I have a Picker Item with `value={10}`, using 2.5.1, the value will be set to `10` when I select that, but starting with 2.6.x, it will be set to `"10"` which is a substantial breaking change.

**Comment by stevenpal** [comment]
Experiencing the same issue with 2.6.1 on iOS (Android works as expected).  Reverting to 2.5.1 for now, which appears compatible with Expo 50 so far (for the Expo users out there).

**Comment by elkinjosetm** [comment]
We are having the same issue in our project, we'll need to revert until a fix is released

**Comment by swrobel** [comment]
@WoLewicki since you worked on [the PR](https://github.com/react-native-picker/picker/pull/456) that seems to have introduced this bug, could you take a look at this?

**Comment by WoLewicki** [comment]
Yeah, I can see that, as mentioned here: https://github.com/react-native-picker/picker/pull/456/files/624af21c3f1fa6212d5f13dfb7f21de8e43ba3af#r1526543524, this makes the value always a `string`. It's like this due to `codegen` not accepting mixed types :disappointed: We would need to change this type to `UnsafeMixed` and parse it correctly on the native side. It is doable, but right now I have little time to do it unfortunately. I took the same approach here: https://github.com/software-mansion/react-native-svg/pull/2238 if someone is willing to contribute. If not, I'll try to find an hour to do it.

**Comment by swrobel** [comment]
@WoLewicki is there any chance you could take another look at this?

**Comment by andrejpavlovic** [comment]
I added this patch to my repo to make sure I'm always passing in strings.

`react-native-picker-select+9.3.1.patch`
```patch
diff --git a/node_modules/react-native-picker-select/index.d.ts b/node_modules/react-native-picker-select/index.d.ts
index cac5c68..38aa318 100644
--- a/node_modules/react-native-picker-select/index.d.ts
+++ b/node_modules/react-native-picker-select/index.d.ts
@@ -11,7 +11,7 @@ import { PickerProps } from '@react-native-picker/picker/typings/Picker';
 
 export interface Item {
     label: string;
-    value: any;
+    value: string;
     key?: string | number;
     color?: string;
     testID?: string;
@@ -68,9 +68,9 @@ type CustomTouchableWrapperProps = Omit<TouchableOpacityProps, 'onPress'>;
 // 'testID' and 'activeOpacity' are also used, but can be overwritten safely
 
 export interface PickerSelectProps {
-    onValueChange: (value: any, index: number) => void;
+    onValueChange: (value: string, index: number) => void;
     items: Item[];
-    value?: any;
+    value?: string;
     placeholder?: Item | {};
     disabled?: boolean;
     itemKey?: string | number;
```

**Comment by WoLewicki** [comment]
Can you check if https://github.com/react-native-picker/picker/pull/600 fixes the issue? Also, does it happen on Android too?
